### ISSUE TITLE
[P2][Sourcing][Accessibility]tree table : tree nodes should be operable using left and right arrow keys

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10261]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
As per w3 standards, tree nodes should be expandable or collapsible using left and right arrow keys, and also using the same set of keys to traverse between table cells

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
Angular 15
0.43.0-rc.16

#### If this is a bug, please provide steps for reproducing it.
replicable on https://sap.github.io/fundamental-ngx/#/platform/table/basic#tree

#### Is there anything else we should know?
please see keyboard support in https://www.w3.org/WAI/ARIA/apg/patterns/treegrid/examples/treegrid-1/

**Comment by droshev** [comment]
ng15

**Comment by g-cheishvili** [comment]
> Hello, This issue is still not fixed has the tree nodes cannot be opened using left/right arrow keys and this needs to be fixed.
> 
> @rvamudha please reopen this ticket.
> 
> Regards, Sagar

What do you mean it is not working?

https://github.com/SAP/fundamental-ngx/assets/28543391/8031bf22-9442-425f-b485-16b866a6acb2

**Comment by droshev** [comment]
@I543348 We would need to remove the row focus as per recommendation from our a11y experts. In that page this is just an example. (treegrid example shows a row focus. The grid examples don’t)
